CMakeLists.txt

```diff
@@ -461,6 +473,7 @@ macro(nebula_link_libraries target)
         dl
         ${GETTIME_LIB}
         -pthread
+        --coverage
```

Comment:
Do not always link against libgcov. And, append it to `CMAKE_EXE_LINKER_FLAGS` only if enabled.

Reply:
> Do not always link against libgcov. And, append it to `CMAKE_EXE_LINKER_FLAGS` only if enabled.

The library will not add to executable if not symbol used (without --coverage option when compile sources in this case).

Reply:
Then make it explicitly.